docs/adding_packages/conanfile_attributes.md

```diff
@@ -57,6 +57,18 @@ The license attribute is a mandatory field which provides the legal information
   * `LicenseRef-` as a prefix, followed by the name of the library. For example:`LicenseRef-libfoo-public-domain`
 * If the library makes no mention of a license and the terms of use - it **shall not be accepted in ConanCenter** , even if the code is publicly available in GitHub or any other platforms.
 
+In case the license changes in a new release, the recipe should update the license attribute accordingly:
+
+```python
+class LibfooConan(ConanFile):
+    license = "MIT" # old license
```

Comment:
I believe when I was talking to @prince-chrismc about this in the past, he recommended placing both in the `license` field here so that they would show up in `conan inspect`.

Reply:
That's true, `conan inspect` does not run `configure` so it lies about the license. I gonna talk to Conan team in case we can do something, otherwise, let's keep both.

Reply:
@jwillikers I talked to the team and a new idea came up, please, check the new commit to see if it's clear enough, otherwise, we can improve the description.

Reply:
That looks great, thank you!